src/useFormo.ts

```diff
@@ -80,25 +77,41 @@ type FormOptions<
   }
 ];
 
+type ArrayFieldValue<A extends unknown[]> = A & { __isFieldArray: true };
+
+export function subForm<A extends unknown[]>(
+  arrayFieldValue: A
+): ArrayFieldValue<A> {
+  (arrayFieldValue as any).__isFieldArray = true;
+  return arrayFieldValue as any;
+}
```

Comment:
hackish, but changing the type / values also "structurally" and this change be reflected internally resulted in too many changes and hard to track & fix for regressions